server/client.go

```diff
@@ -585,8 +624,8 @@ func (c *client) initClient() {
 	c.cid = atomic.AddUint64(&s.gcid, 1)
 
 	// Outbound data structure setup
-	c.out.sz = startBufSize
 	c.out.sg = sync.NewCond(&(c.mu))
+	c.out.nb = make(net.Buffers, 0, nbStartCount)
```

Comment:
We want to optimize to no more than 8 yes?

Reply:
8 comes from the quick path for `writev`, but we can still save on some potential re-allocations by ensuring we have a reasonable amount of space after we've exceeded those 8 values. This is just an array of `[][]byte` references so they are compact, even with 128 of them.

Reply:
Yes bit what I am saying is that for a call into the kernel let's limit to 8, so that would most likely be 8*64k or 256k.. WDYT?

Reply:
I've done some researching on this and it appears there's no benefit, when heavily loaded we end up just making more syscalls so the time lost to context switching is worse than the gain from hitting the `writev` fast path. :-(